pkg/cmd/admin/router/router.go

```diff
@@ -755,6 +758,11 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 					}
 				}
 			}
+			if len(defaultCert) == 0 {
+				// When a user does not provide the default cert (pem), create one via a Service annotation
+				// The secret generated by the service annotaion contains a tls.crt and tls.key
+				t.Annotations = map[string]string{"service.alpha.openshift.io/serving-cert-secret-name": name + "-certs"}
```

Comment:
Can we get the same cert name somehow?  Either put it somewhere common, or make the code that generates it a private function that takes the name as an argument.